titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 5_
```python
import seaborn as sns
import matplotlib.pyplot as plt

# Load the Titanic dataset
titanic = sns.load_dataset('titanic')
```
A new comment to mark as resolved

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/10)._

Reply:
her's a reply to the comment